compute/cloud-client/src/test/java/compute/deleteprotection/DeleteProtectionIT.java

```diff
@@ -42,13 +39,10 @@
 @RunWith(JUnit4.class)
 @Timeout(value = 10, unit = TimeUnit.MINUTES)
 public class DeleteProtectionIT {
-
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
-  private static final String ZONE = getZone();
+  private static final String ZONE = "asia-south1-a";
```

Comment:
Is there a specific reason to hardcode the zone here?

Reply:
The reason is to create resources in chosen zone and if test fails , then it will be deleted next test run. If it has 3 option for zone, it can be deleted at third try. And if we have this getZone for most tests, we can have a lots of resources created.